circle.yml

```diff
@@ -15,10 +15,11 @@ dependencies:
     - "~/.cabal"
 
   override:
+    - rm -rf /home/ubuntu/.go_workspace/src
     - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
     - cp -r /home/ubuntu/git-town /home/ubuntu/.go_workspace/src/github.com/Originate/gt
-    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && go install
-    - rm -rf /home/ubuntu/.go_workspace/src/github.com/Originate/gt
+    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && go install && go get gopkg.in/alecthomas/gometalinter.v1
```

Comment:
This is necessary to install the binary. I'm not sure it's the correct way, but at least it works.